infoblox_client/objects.py

```diff
@@ -322,13 +327,60 @@ def get_v6_class(cls):
         return cls
 
 
+class EA(object):
+    """Extensible Attributes
+
+    This class represents extensible attributes (EA).
+    Converts EAs into format suitable for NIOS (to_dict)
+    and builds EA class from NIOS reply (from_dict).
+    """
+
+    def __init__(self, ea_dict=None):
+        """Optionally accept EAs as a dict on init
+
+        Expected EA format is {ea_name: ea_value}
+        """
+        if ea_dict is None:
+            ea_dict = {}
+        self._ea_dict = ea_dict
+
+    def __repr__(self):
+        eas = ()
+        if self._ea_dict:
+            eas = ("{0}={1}".format(name, self._ea_dict[name])
+                   for name in self._ea_dict)
+        return "EAs:{0}".format(','.join(eas))
+
+    @classmethod
+    def from_dict(cls, eas_from_nios):
+        if not eas_from_nios:
+            return
+        return cls({name: eas_from_nios[name]['value']
+                    for name in eas_from_nios})
+
+    def to_dict(self):
```

Comment:
Alright, I was wrong, looks like it is notion to use it as class method all over the project.